x-pack/plugins/actions/server/plugin.ts

```diff
@@ -246,9 +246,10 @@ export class ActionsPlugin implements Plugin<Promise<PluginSetupContract>, Plugi
   }
 
   private createRouteHandlerContext = (
+    coreSetup: CoreSetup,
```

Comment:
I believe the call `core.http.registerRouteHandlerContext(...)` above in this file will need to pass in this new argument.